src/main/resources/db/migration/liquibase/data/latest/widgets/base/clientListHeader.widget.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "clientListHeader",
+  "title": "Client List",
+  "type": "Header",
+  "bc": "client",
+  "showCondition": [],
+  "fields": [],
+  "axisFields": [],
```

Comment:
Check the schema https://github.com/tesler-platform/tesler-schema/blob/b7b6a9b4874d82cd2a010f6370b275fe33375121/src/files/WidgetMeta.ts#L25

We only need mandatory fields, and we definitely do not want legacy fields which are not even in the schema to be in out demo project

Reply:
@Dergash checked